app/Jobs/RequeuePendingQsBatchesJob.php

```diff
@@ -31,7 +31,10 @@ public function handle(): void
 
             $threshold = Carbon::now()->subtract(new \DateInterval($this->pendingTimeout));
             QsBatch::where([['pending_since', '<>', null], ['pending_since', '<', $threshold]])
-                ->update(['pending_since' => null, 'done' => 0]);
```

Comment:
The current state would retry a batch that makes the updater fail with an unhandled exception (i.e. malformed payload) forever, so requeueing needs to increment the couter as well.